main.css

```diff
@@ -1,26 +1,29 @@
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;400;700&display=swap');
 
 :root {
```

Comment:
This is a huge file. It will be hard to maintain, break it into small files based on components and import them into one place.

Reply:
Thanks for the suggestion, will refactor it soon.